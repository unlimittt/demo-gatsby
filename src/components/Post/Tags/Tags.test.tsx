import React from "react"
import Tags from "./Tags"
import Enzyme, { shallow } from "enzyme"
import Adapter from "enzyme-adapter-react-16"

Enzyme.configure({ adapter: new Adapter() })

describe("Tags", () => {
  it("renders correctly", () => {
    const props = {
      tags: ["test_0", "test_1"],
      tagSlugs: ["/test_0", "/test_1"],
    }

    const wrapper = shallow(<Tags {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
