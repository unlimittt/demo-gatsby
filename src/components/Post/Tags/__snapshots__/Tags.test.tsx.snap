// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags renders correctly 1`] = `
<div
  className="tags"
>
  <ul
    className="tags__list"
  >
    <li
      className="tags__list-item"
      key="test_0"
    >
      <mockConstructor
        className="tags__list-item-link"
        to="/test_0"
      >
        test_0
      </mockConstructor>
    </li>
    <li
      className="tags__list-item"
      key="test_1"
    >
      <mockConstructor
        className="tags__list-item-link"
        to="/test_1"
      >
        test_1
      </mockConstructor>
    </li>
  </ul>
</div>
`;
